### ISSUE TITLE
找到一个字体管理插件：FontManager

### ISSUE THREAD

**Issue by Zacharia2** [tiddly-gittly/TiddlyWiki-CPL#55]
主页：https://fontmanager.tiddlyhost.com/

下载链接：https://fontmanager.tiddlyhost.com/#How%20to%20install%20the%20font%20manager

![image](https://user-images.githubusercontent.com/32425955/179186483-d87fdc2c-5f43-46a7-9325-de4175e2e1aa.png)

**Comment by Gk0Wk** [comment]
暂时保留，理由：这个作者大哥怎么都不打包一下，导出的 JSON 文件不是插件格式，而是多个 tid 的打包，不是插件。

等有时间来打包一下。

**Comment by linonetwo** [comment]
No, only a github release link is needed. like https://github.com/Gk0Wk/TW5-CodeMirror-Enhanced/releases/tag/v0.1.4 have 

`https://github.com/Gk0Wk/TW5-CodeMirror-Enhanced/releases/latest/download/__plugins_Gk0Wk_codemirror-themes.json`

**Comment by Telumire** [comment]
@linonetwo done! https://github.com/Telumire/Tiddlywiki-Fonts-Plugin/releases/latest/download/plugin_Telumire_Fonts.json

**Comment by Zacharia2** [comment]
@Telumire OK，Thank you。
